backend/app/email_templates/security_login.html

```diff
@@ -0,0 +1,7 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+Someone recently logged into your account on Flathub using the <strong>{{ provider }}</strong> account <strong>{{ login }}</strong>. If it wasn't you, please contact us immediately.
```

Comment:
Wondering if we should say how? Not sure the mail address this mail is coming from is going to be one that's getting read? @barthalion